packages/terra-table/src/proptypes/headerCellShape.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+
+const headerCellShape = PropTypes.shape({
+  /**
+   * The id of the header cell for the row cells to set within their "headers" prop.
+   */
+  id: PropTypes.string.isRequired,
+  /**
+   * The react key to apply to the cell.
+   */
+  key: PropTypes.string.isRequired,
+  /**
+   * Content to be displayed for the column header.
+   */
+  children: PropTypes.node,
+  /**
+   * Whether or not the sort indicator is descending.
+   */
+  isSortDesc: PropTypes.bool,
+  /**
+   * Whether or not the column is to be marked as a sorted column.
+   */
+  isSortActive: PropTypes.bool,
+  /**
+   * The associated metaData to be provided in the onSelect callback.
```

Comment:
Should this be `onCellAction` and `onSortAction`?

Reply:
Updated.